prog/mutation_test.go

```diff
@@ -152,6 +152,49 @@ func TestMutateArgument(t *testing.T) {
 	}
 }
 
+func TestSizeMutateArg(t *testing.T) {
```

Comment:
Could we fold all of this test into TestSizeGenerateConstArg? They seem to have lots of logic in common. So that test could test both generation and mutation.